<launch>
      <arg name="drone_id" default="1" />
      <arg name="t_final_in" default="60.0" />
      <arg name="trajectory" default="long_line_slow" />
      <arg name="rc_trig" default="0" />

      <node pkg="bridge_px4" type="GCSRogue" name ="GCSRogue" output="screen">
            <param name="auto_rc_trig" value="$(arg rc_trig)"/>
            <param name="traj_id" value="$(find bridge_px4)/trajectories/$(arg trajectory).csv"/>
            <param name="t_final" value="$(arg t_final_in)" />
      </node>
</launch>
